packages/teams-js/src/public/liveShareHost.ts

```diff
@@ -226,7 +248,13 @@ export class LiveShareHost {
   public setFluidContainerId(containerId: string): Promise<liveShare.IFluidContainerInfo> {
     ensureSupported();
     return new Promise<liveShare.IFluidContainerInfo>((resolve) => {
-      resolve(sendAndHandleSdkError('interactive.setFluidContainerId', containerId));
+      resolve(
+        sendAndHandleSdkErrorWithVersion(
+          getApiVersionTag(interactiveTelemetryVersionNumber, ApiName.Interactive_SetFluidContainerId),
+          'interactive.setFluidContainerId',
```

Comment:
This is probably a bigger question than what should be handled in this PR, so it absolutely should not block you from checking in, but should the message being passed and the `ApiName` constant being used just be the same? In most (all?) cases, they're literally the same string. And representing them twice opens the possibility of unintentional divergence.

Reply:
I have thought of this question and my answer to this is: Ideally, yes. That is to say the value of `funcName` as a parameter in SDK request should be EXACTLY the same as value of ApiVersionTag and developers don't have to write the same string twice in the future.

However, for now, due to some constraints, i.e. tech debt, openLink special scenarios, we cannot make it. But what I will suggest to do after this project is to use ApiName as the value of `funcName` to make sure that API keeps the `funcName` to be exactly the same as `apiVersionTag` so that in the future, we could remove `apiVersionTag` and use `funcName` for telemetry (at that moment the `funcName`s are API names).

Does that make sense to you? @AE-MS
(@TrevorJoelHarris tagged you for my suggestion in the future)